packages/manager/apps/dedicated/client/app/dedicated/server/details/server.html

```diff
@@ -43,7 +44,7 @@
             data-href="{{:: $ctrl.getTabItemUrl('ftpBackup') }}"
             active="$ctrl.getTabItemUrl('ftpBackup') === $ctrl.currentActiveLink()"
             data-disabled="server.isExpired"
-            data-ng-if="worldPart !== 'US'"
+            data-ng-if="worldPart !== 'US' && $ctrl.features.isFeatureAvailable('dedicated-server:backup')"
```

Comment:
Both `worldPart` and `region` (from `feature-availability` point of view) are the same.

Why not prefer updating the feature rule `dedicated-server:backup` to match the corresponding regions supported?
In that way, it will allow us to remove those `ngIf`.

Reply:
Nice idea. This is done :)